pages/common/composer.md

```diff
@@ -0,0 +1,31 @@
+# composer
+
+> Dependency Manager for PHP.
+
+- Interactively create a composer.json file:
+
+`composer init`
+
+- Create new project from an existing package:
+
+`composer create-project {{project_name}} {{target_path}} {{version}}`
```

Comment:
"path" may be ambiguous. Using the format {{path/to/project_location}} (or something alike) may make it clearer that the token expects a filesystem path.

Also, I wonder if {{version}} is required, and if so, whether there's a default version if the parameter isn't passed.